repour/adjust/util.py

```diff
@@ -126,7 +126,7 @@ def get_extra_parameters(extra_adjust_parameters):
     """
```

Comment:
Shouldn't the comment be updated? This isn't just for PME any more ?

Reply:
I just updated the lines with parameters, not sure if anything else should be changed too about PME mentioned there.